servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/H2ClientParentConnectionContext.java

```diff
@@ -93,7 +93,7 @@
 
 final class H2ClientParentConnectionContext extends H2ParentConnectionContext {
     private H2ClientParentConnectionContext(Channel channel, HttpExecutionContext executionContext,
-                                            FlushStrategy flushStrategy, @Nullable Long idleTimeoutMs,
+                                            FlushStrategy flushStrategy, Long idleTimeoutMs,
```

Comment:
At 0.43 you may want to switch the signature to `long` since users will have to recompile. (Or add a deprecate now and a new `long` overload)